migrations.md

```diff
@@ -301,6 +301,40 @@ Modifier  |  Description
 `->generatedAs($expression)`  |  Create an identity column with specified sequence options (PostgreSQL)
 `->always()`  |  Defines the precedence of sequence values over input for an identity column (PostgreSQL)
 
+#### Default Expressions
+
+The `default` modifier can take either a value or an `\Illuminate\Database\Query\Expression`. Using an `Expression` will prevent wrapping the value in quotes and allow you to use database specific functions.
+
+One place this is particularly useful is assigning default values to JSON columns:
+
+```php
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFlightsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->json('movies')->default(new Expression('(JSON_ARRAY())'));
+            $table->timestamps();
+        });
+    }
+}
+```
```

Comment:
You probably want to indent the code block here like the other examples. instead of using the ``` signs.